syntaxes/nim.json

```diff
@@ -148,7 +148,7 @@
     },
     {
       "comment": "Keywords that affect program control flow or scope.",
-      "match": "\\b(block|break|case|continue|do|elif|else|end|except|finally|for|if|raise|return|try|when|while|yield)\\b",
+      "match": "\\b(block|break|case|continue|do|elif|else|end|except|finally|for|if|raise|return|try|when|while|yield|(static(?= *:))|of)\\b",
```

Comment:
Shouldn't only the static block one count here? Wouldn't that also match it in types for procedure parameters?

Reply:
ah i thought i took care of that but i'll look more into the places where static is used and make sure it doesn't match those as well

Reply:
ok so it matches a colon after the static. a type like `static string` would have a colon before not after